### PR TITLE
docs: HTTPChaos TLS support

### PR DESCRIPTION
**Pull request by Andrewmatilde**
Signed-off-by: andrewmatilde <davis6813585853062@outlook.com>

### CODE REVIEW

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -139,6 +139,11 @@ Common fields are meaningful when the `target` of fault injection is `Request` o
 | `patch.body.value` | string | Specifies the fault of the request body or response body with patch faults. |  | no | "{"foo": "bar"}" |
 | `duration` | string | Specifies the duration of a specific experiment. |  | yes | 30s |
 | `scheduler` | string | Specifies the scheduling rules for the time of a specific experiment. |  | no | 5 \* \* \* \* |
+| `tls.secretName` | string | SecretName represents the name of required secret resource | | no | "http-tls-scr" |
+| `tls.secretNamespace` | string | SecretNamespace represents the namespace of required secret resource,should be the same with deployment/chaos-controller-manager in most cases | | no | "chaos-mesh" |
+| `tls.certName` | string | CertName represents the data name of cert file in secret, `tls.crt` for example | | no | "tls.crt" |
+| `tls.KeyName` | string | KeyName represents the data name of key file in secret, `tls.key` for example | | no | "tls.key" |
+| `tls.caName` | string | CAName represents the data name of ca file in secret, `ca.crt` for example | | no | "ca.crt" |
```
Comment (Oreoxmt) [inline docs/simulate-http-chaos-on-kubernetes.md:146]:
```suggestion
| `tls.secretName` | string | Specifies the name of the required secret resource. | | no | "http-tls-scr" |
| `tls.secretNamespace` | string | Specifies the namespace of the required secret resource. In most cases, it should be the same with `deployment/chaos-controller-manager`. | | no | "chaos-mesh" |
| `tls.certName` | string | Specifies the data name of the cert file in secret. | | no | "tls.crt" |
| `tls.KeyName` | string | Specifies the data name of the key file in secret. | | no | "tls.key" |
| `tls.caName` | string | Specifies the data name of the CA file in secret. | | no | "ca.crt" |
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
@Andrewmatilde Could you please invite a tech review? Thanks

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 1 above)
Comment (Andrewmatilde) [reply docs/simulate-http-chaos-on-kubernetes.md]: I have added it to secretName/

**Comment by cwen0** [comment]
@Andrewmatilde Can you add this doc to v2.5 version?

File: docs/simulate-http-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -246,12 +250,76 @@ spec:
...
+## TLS
+
+To inject faults inside connection base on TLS, user should use TLS mode. Our proxy play a proxy role here, so in TLS people both need to act as a remote server with a trustful CA , but also need to act as a client trust the server with some ca.
+
+So in the secret data blow user need to create its' TLS keys & CA & CRT on their own.
+
+```
+{
+	"tls.certName":cert,
+    "tls.KeyName":key,
```
Comment (cwen0) [inline docs/simulate-http-chaos-on-kubernetes.md:268]: format this

File: versioned_docs/version-2.5.0/simulate-http-chaos-on-kubernetes.md [hunk 3]
```diff
@@ -251,6 +256,70 @@ spec:
...
+## TLS
+
+To inject faults inside connection base on TLS, user should use TLS mode. Our proxy play a proxy role here, so in TLS people both need to act as a remote server with a trustful CA , but also need to act as a client trust the server with some ca.
+
+So in the secret data blow user need to create its' TLS keys & CA & CRT on their own.
+
+```
+{
+	"tls.certName":cert,
+    "tls.KeyName":key,
```
Comment (cwen0) [inline versioned_docs/version-2.5.0/simulate-http-chaos-on-kubernetes.md:268]: ditto

**Comment by cwen0** [comment]
@Andrewmatilde PTAL

File: docs/simulate-http-chaos-on-kubernetes.md (hunk 2 above)
Comment (Andrewmatilde) [reply docs/simulate-http-chaos-on-kubernetes.md]: Fixed.

File: versioned_docs/version-2.5.0/simulate-http-chaos-on-kubernetes.md (hunk 3 above)
Comment (Andrewmatilde) [reply versioned_docs/version-2.5.0/simulate-http-chaos-on-kubernetes.md]: Fixed.

**Review by cwen0 — APPROVED** [review]

**Pull request merged**